docs: select $N from csv (#11813)
datafuselabs/databend: docs/doc/12-load-data/00-transform/05-querying-stage.md

### $<col_position>

> You can use column positions when you SELECT FROM a staged file. At present, the feature to utilize column positions for SELECT operations from staged files is limited to Parquet, NDJSON, and CSV formats.

Reviewer: ```suggestion
You can use column positions when you SELECT FROM a staged file. At present, the feature to utilize column positions for SELECT operations from staged files is limited to Parquet, NDJSON, CSV and TSV formats.
```

They are explained in [Create Stage](../../14-sql-commands/00-ddl/40-stage/01-ddl-create-stage.md).

## Limitations

When querying a staged file, the following limitations are applicable in terms of format-specific constraints:

- Selecting all fields with the symbol * is only supported for Parquet files.

> - When selecting from a CSV or TSV file, it is necessary for all fields to be of the String data type, and if column positions are used in the SELECT statement, the number of fields in the file must not exceed N+1000. In other words, if the statement is `SELECT $2 FROM @my_stage (FILES=>('sample.csv'))`, the sample.csv file can have a maximum of 1,002 fields.

Reviewer: ```suggestion
- When selecting from a CSV or TSV file,  all fields are parsed as  the String type, and only column positions can be used in the SELECT statement, the number of fields in the file must not exceed (max of N) + 1000. In other words, if the statement is `SELECT $2 FROM @my_stage (FILES=>('sample.csv'))`, the sample.csv file can have a maximum of 1,002 fields.
```
